README.md

```diff
@@ -32,3 +34,5 @@ If you use iDFlakies, please cite our corresponding [ICST paper](http://winglam2
     year      = {2019},
 }
 ```
+
+Additional information about the projects analyzed in the paper can be found [here](https://sites.google.com/view/flakytestdataset/home)
```

Comment:
Please add a period and move this sentence to be directly under the iDFlakies section so that people see this easily. You can just say "Additional information about iDFlakies can be found [here]..."